test/onnx/test_operators.py

```diff
@@ -339,7 +339,8 @@ class MyModule(Module):
             def forward(self, x):
                 return MyFun.apply(x)
 
-        self.assertONNX(MyModule(), x)
+        self.assertONNX(MyModule(), x,
+                        operator_export_type=torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK)
```

Comment:
what happens without adding `operator_export_type`?

Reply:
`RuntimeError: ONNX export failed: Couldn't export operator aten::ATen` is raised, because a `ATen` op is created and export is on `ONNX` only mode

Reply:
isee, because previously it was onnx::aten. thx